peer/log.go

```diff
@@ -68,6 +68,8 @@ func invSummary(invList []*wire.InvVect) string {
 			return fmt.Sprintf("tx %s", iv.Hash)
 		case wire.InvTypeFilteredBlock:
 			return fmt.Sprintf("filtered block %s", iv.Hash)
+		case wire.InvTypeMix:
+			return fmt.Sprintf("mix message %s", iv.Hash)
```

Comment:
Should also hit the case when there are multiple requests in the same inv below.  e.g.

```diff
diff --git a/peer/log.go b/peer/log.go
index d142f5d9..aa856dec 100644
--- a/peer/log.go
+++ b/peer/log.go
@@ -74,18 +74,21 @@ func invSummary(invList []*wire.InvVect) string {
        }
 
        // More than one inv item.
-       var numTxns, numBlocks uint64
+       var numTxns, numBlocks, numMixes uint64
        for _, iv := range invList {
                switch iv.Type {
                case wire.InvTypeTx:
                        numTxns++
                case wire.InvTypeBlock:
                        numBlocks++
+               case wire.InvTypeMix:
+                       numMixes++
                }
        }
 
-       diff := uint64(invLen) - (numTxns + numBlocks)
-       return fmt.Sprintf("txns %d, blocks %d, other %d", numTxns, numBlocks, diff)
+       diff := uint64(invLen) - (numTxns + numBlocks + numMixes)
+       return fmt.Sprintf("txns %d, blocks %d, mixmsgs %d, other %d", numTxns,
+               numBlocks, numMixes, diff)
 }
```

Reply:
i had the same diff, word for word.